partcad/src/partcad/sketch_factory_python.py

```
@@ -25,13 +25,14 @@ def __init__(
         config,
         can_create=False,
         python_version=None,
+        extension=".py",
```

Ruff (0.8.2)

Missing type annotation for function argument `extension`

(ANN001)